table/column.go

```diff
@@ -183,6 +183,7 @@ func CastValue(ctx sessionctx.Context, val types.Datum, col *model.ColumnInfo) (
 		return casted, err
 	}
 
+	// TODO: we may need to handle non-binary collation.
```

Comment:
We don't need to handle non-binary colltion here, they are handled in other places.

Reply:
Oh it's an outdated comment